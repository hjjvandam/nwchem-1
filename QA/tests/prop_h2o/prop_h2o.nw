scratch_dir /scratch/mdupuis
title
  h2o
geometry units au nocenter 
o  .00000000     -.00000000      .27143053
h  .00000000     1.88042809     -.83572213
h  .00000000    -1.48042809    -1.33572213
end
basis segment noprint
  H library sto-3g
  O library sto-3g
end
 charge 1
 scf
   rohf
   doublet
 end
 dft
   mult 2
 end
property
  dipole
  quadrupole
  octupole
  esp
  efield
  efieldgrad
  electrondensity
end
task scf
task property
eof
