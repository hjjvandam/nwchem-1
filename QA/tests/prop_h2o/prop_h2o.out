 argument  1 = prop_h2o.nw
                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 2.1
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                                         
                                         
 
 
                         COPYRIGHT (C) 1994, 1995, 1996, 1997
                        Pacific Northwest National Laboratory,
                             Battelle Memorial Institute.
 
                            >>> All Rights Reserved <<<
 
 
                                    DISCLAIMER
                                    ----------

            This material was prepared as an account of work sponsored
            by an agency of the United States Government.  Neither the
            United States Government nor the United States Department
            of Energy, nor Battelle, nor any of their employees, MAKES
            ANY WARRANTY, EXPRESS OR IMPLIED, OR ASSUMES ANY LEGAL
            LIABILITY OR RESPONSIBILITY FOR THE ACCURACY, COMPLETENESS,
            OR USEFULNESS OF ANY INFORMATION, APPARATUS, PRODUCT,
            SOFTWARE, OR PROCESS DISCLOSED, OR REPRESENTS THAT ITS USE
            WOULD NOT INFRINGE PRIVATELY OWNED RIGHTS.


                                    LIMITED USE
                                    -----------

            This software (including any documentation) is being made
            available to you for your internal use only, solely for use
            in performance of work directly for the U.S. Federal
            Government or work under contracts with the U.S. Department
            of Energy or other U.S. Federal Government agencies.  This
            software is a version which has not yet been evaluated and
            cleared for commercialization.  Adherence to this notice
            may be necessary for the author, Battelle Memorial
            Institute, to successfully assert copyright in and
            commercialize this software. This software is not intended
            for duplication or distribution to third parties without
            the permission of the Manager of Software Products at
            Pacific Northwest National Laboratory, Richland,
            Washington, 99352.


                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were produced with
            Government support under Contract Number DE-AC06-76RLO-1830
            awarded by the United States Department of Energy.  The
            Government retains a paid-up non-exclusive, irrevocable
            worldwide license to reproduce, prepare derivative works,
            perform publicly and display publicly by or for the
            Government, including the right to distribute to other
            Government contractors.


              Job information
              ---------------

    hostname  = hondo
    program   = nwchem
    date      = Tue Jun 24 13:10:32 1997

    compiled  = Tue Jun 24 13:09:57 PDT 1997
    source    = /scratch/mdupuis/nwchem
    input     = prop_h2o.nw
    prefix    = prop_h2o.
    data base = ./prop_h2o.db
    status    = startup
    nproc     =    1


            Memory information
            ------------------

    heap      =    1572865 doubles
    stack     =    1572865 doubles
    global    =    3145728 doubles (allocated separately from heap & stack)
    total     =    6291458 doubles
    verify    = yes
    hardfail  = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = /scratch/mdupuis
 
 
 
                                NWCHEM Input Module
                                -------------------
 
 
                                       h2o
                                       ---
 
 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 o                    8.0000      .00000000      .00000000      .27143053
    2 h                    1.0000      .00000000     1.88042809     -.83572213
    3 h                    1.0000      .00000000    -1.48042809    -1.33572213
 
      Atomic Mass 
      ----------- 
 
      o                 15.994910
      h                  1.007825
 

 Effective nuclear repulsion energy (a.u.)       7.6215870097

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
      .0000000000      .4000000000     -.0000000200
 
                                 NWChem SCF Module
                                 -----------------
 
 
                                        h2o
 
 

  ao basis        = "ao basis"
  functions       =     7
  atoms           =     3
  closed shells   =     4
  open shells     =     1
  charge          =   1.00
  wavefunction    = ROHF
  input vectors   = atomic
  output vectors  = ./prop_h2o.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           sto-3g                  1        1   1s
 O                           sto-3g                  3        5   2s1p



 Forming initial guess at        .3s

  Calling module = scf
 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -74.71095592

 Renormalizing density from    10.00 to    9

 #quartets = 1.200D+02 #integrals = 2.600D+02 #direct =   .0% #cached =100.0%


 Integral file          = /scratch/mdupuis/prop_h2o.aoints.0
 Record size in doubles =  32769        No. of integs per rec  =  32768
 Max. records in memory =      2        Max. records in file   =   6289
 No. of bits per label  =      8        No. of bits per value  =     32

 
      Non-variational initial energy
      ------------------------------

 Total energy =     -69.409736
 1-e energy   =    -107.019611
 2-e energy   =      29.988288
 HOMO         =       -.932253
 LUMO         =       -.193346
 

 Starting SCF solution at        .5s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           20
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -74.5608462837  8.53D-01  6.74D-01       .5
                 2      -74.6423551686  1.78D-01  1.44D-01       .6
                 3      -74.6435900938  3.04D-02  2.46D-02       .7
                 4      -74.6436235209  7.26D-04  5.17D-04       .7
                 5      -74.6436236312  3.42D-06  2.52D-06       .8


       Final ROHF results 
       ------------------ 

         Total SCF energy =    -74.643623631219
      One-electron energy =   -114.162756013554
      Two-electron energy =     31.897545372614
 Nuclear repulsion energy =      7.621587009721

        Time for solution =       .3s


             Final eigenvalues
             -----------------

              1      
    1  -20.9500
    2   -1.7154
    3   -1.0346
    4    -.9608
    5    -.6060
    6    -.0349
    7     .0444
 
                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-1.715450D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2       .931848  1 O  s                  1      -.245686  1 O  s          
 
 Vector    3  Occ=2.000000D+00  E=-1.034632D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4       .727920  1 O  py                 6       .349576  2 H  s          
     7      -.348624  3 H  s          
 
 Vector    4  Occ=2.000000D+00  E=-9.607612D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5       .826448  1 O  pz                 2       .374700  1 O  s          
     7      -.240543  3 H  s                  6      -.239098  2 H  s          
 
 Vector    5  Occ=1.000000D+00  E=-6.059876D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.000000  1 O  px         
 
 Vector    6  Occ=0.000000D+00  E=-3.489194D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7       .738294  3 H  s                  6       .726531  2 H  s          
     5       .615219  1 O  pz                 2      -.613297  1 O  s          
 
 Vector    7  Occ=0.000000D+00  E= 4.440431D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4       .787308  1 O  py                 6      -.768025  2 H  s          
     7       .755632  3 H  s          
 
 
  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 O    8     7.84   2.00  1.91  3.93
    2 H    1      .58    .58
    3 H    1      .58    .58
 
 Mulliken analysis of the open-shell density
 -------------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 O    8     1.00    .00   .00  1.00
    2 H    1      .00    .00
    3 H    1      .00    .00
 
     Multipole analysis of the density
     ---------------------------------
 
     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      1.000000     -1.000000     10.000000
 
     1   1 0 0       .000000       .000000       .000000
     1   0 1 0       .199133       .000000       .400000
     1   0 0 1     -1.082446      -.271431       .000000
 
     2   2 0 0     -3.543907      -.891272       .000000
     2   1 1 0       .000000       .000000       .000000
     2   1 0 1       .000000       .000000       .000000
     2   0 2 0     -1.084717      -.297091      5.727677
     2   0 1 1       .208226       .000000       .405925
     2   0 0 2     -2.736100      -.370765      3.071981
 

 Parallel integral file used       1 records with       0 large values

 
 
                                NWCHEM Input Module
                                -------------------
 
 
                                    Properties
                                    ----------
                              NWCHEM HND_PROP Module
                              ----------------------
 
 
                                        h2o
 

 ----- property selection -----
 nodip = 0 noqdp = 0 nootp = 0 nopop = 1 nospin= 1 noloc = 1 nodpl = 1 nodhp = 1 nod2hp= 1
 nofpl = 1 nofhp = 1 nof2hp= 1 nosos = 1 noelp = 0 noelf = 0 noelfg= 0 noden = 0 nonmr = 1 noston= 1

          ---------------------
          Atomic weights (a.u.)
          ---------------------

    1     o                15.99491
    2     h                 1.00782
    3     h                 1.00782

 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs


   Center of mass X =        .0000000 Y =        .0223830 Z =        .1195449

   Dipole moment        1.2149170218 A.U. ( W.R.T. center of mass) 
             DMX         .0000000000 DMXEFC         .0000000000
             DMY         .1767498881 DMYEFC         .0000000000
             DMZ       -1.2019912008 DMZEFC         .0000000000
   -EFC- dipole          .0000000000 A.U.
   Total dipole         1.2149170218 A.U.

   Dipole moment        3.0880343900 Debye(s) ( W.R.T. center of mass) 
             DMX         .0000000000 DMXEFC         .0000000000
             DMY         .4492567995 DMYEFC         .0000000000
             DMZ       -3.0551799820 DMZEFC         .0000000000
   -EFC- dipole          .0000000000 DEBYE(S)
   Total dipole         3.0880343900 DEBYE(S)
 1 a.u. = 2.541766 Debyes 

          ---------------------
          Atomic weights (a.u.)
          ---------------------

    1     o                15.99491
    2     h                 1.00782
    3     h                 1.00782

 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs


 < R**2 > =  16.029717 a.u.  ( 1 a.u. = 0.280023 10**(-16) cm**2 ) 
 ( also called diamagnetic susceptibility ) 

   Center of mass X =        .0000000 Y =        .0223830 Z =        .1195449

   Second moments in atomic units ( W.R.T. center of mass) 
   QMXX       -3.5439071397 QMXXEF         .0000000000 QMXXT       -3.5439071397
   QMYY       -1.0931307814 QMYYEF         .0000000000 QMYYT       -1.0931307814
   QMZZ       -2.4630073575 QMZZEF         .0000000000 QMZZT       -2.4630073575
   QMXY         .0000000000 QMXYEF         .0000000000 QMXYT         .0000000000
   QMXZ         .0000000000 QMXZEF         .0000000000 QMXZT         .0000000000
   QMYZ         .2113251609 QMYZEF         .0000000000 QMYZT         .2113251609

   Second moments in buckingham(s) ( W.R.T. center of mass) 
   QMXX       -4.7662396952 QMXXEF         .0000000000 QMXXT       -4.7662396952
   QMYY       -1.4701636123 QMYYEF         .0000000000 QMYYT       -1.4701636123
   QMZZ       -3.3125256882 QMZZEF         .0000000000 QMZZT       -3.3125256882
   QMXY         .0000000000 QMXYEF         .0000000000 QMXYT         .0000000000
   QMXZ         .0000000000 QMXZEF         .0000000000 QMXZT         .0000000000
   QMYZ         .2842135334 QMYZEF         .0000000000 QMYZT         .2842135334

   Quadrupole moments in atomic units ( W.R.T. center of mass) 
   THXX       -1.7658380703 THXXEF         .0000000000 THXXT       -1.7658380703
   THYY        1.9103264673 THYYEF         .0000000000 THYYT        1.9103264673
   THZZ        -.1444883970 THZZEF         .0000000000 THZZT        -.1444883970
   THXY         .0000000000 THXYEF         .0000000000 THXYT         .0000000000
   THXZ         .0000000000 THXZEF         .0000000000 THXZT         .0000000000
   THYZ         .3169877413 THYZEF         .0000000000 THYZT         .3169877413

   Quadrupole moments in buckingham(s) ( W.R.T. center of mass) 
   THXX       -2.3748950449 THXXEF         .0000000000 THXXT       -2.3748950449
   THYY        2.5692190794 THYYEF         .0000000000 THYYT        2.5692190794
   THZZ        -.1943240345 THZZEF         .0000000000 THZZT        -.1943240345
   THXY         .0000000000 THXYEF         .0000000000 THXYT         .0000000000
   THXZ         .0000000000 THXZEF         .0000000000 THXZT         .0000000000
   THYZ         .4263203001 THYZEF         .0000000000 THYZT         .4263203001

 1 a.u. = 1.344911 Buckinghams = 1.344911 10**(-26) esu*cm**2 

          ---------------------
          Atomic weights (a.u.)
          ---------------------

    1     o                15.99491
    2     h                 1.00782
    3     h                 1.00782

 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs


   Center of mass X =        .0000000 Y =        .0223830 Z =        .1195449

   Third moments in atomic units ( W.R.T. center of mass) 
   RXXX         .0000000000 RXXXEF         .0000000000 RXXXT         .0000000000
   RYYY        1.4198061232 RYYYEF         .0000000000 RYYYT        1.4198061232
   RZZZ       -1.2640525654 RZZZEF         .0000000000 RZZZT       -1.2640525654
   RXXY        -.0513804625 RXXYEF         .0000000000 RXXYT        -.0513804625
   RXXZ         .3471756481 RXXZEF         .0000000000 RXXZT         .3471756481
   RYYX         .0000000000 RYYXEF         .0000000000 RYYXT         .0000000000
   RYYZ       -3.0917579811 RYYZEF         .0000000000 RYYZT       -3.0917579811
   RZZX         .0000000000 RZZXEF         .0000000000 RZZXT         .0000000000
   RZZY        -.7918268428 RZZYEF         .0000000000 RZZYT        -.7918268428
   RXYZ         .0000000000 RXYZEF         .0000000000 RXYZT         .0000000000

   Third moments in 10**(-34) esu*cm**3 ( W.R.T. center of mass) 
   RXXX         .0000000000 RXXXEF         .0000000000 RXXXT         .0000000000
   RYYY        1.0104589802 RYYYEF         .0000000000 RYYYT        1.0104589802
   RZZZ        -.8996110421 RZZZEF         .0000000000 RZZZT        -.8996110421
   RXXY        -.0365668586 RXXYEF         .0000000000 RXXYT        -.0365668586
   RXXZ         .2470807426 RXXZEF         .0000000000 RXXZT         .2470807426
   RYYX         .0000000000 RYYXEF         .0000000000 RYYXT         .0000000000
   RYYZ       -2.2003670540 RYYZEF         .0000000000 RYYZT       -2.2003670540
   RZZX         .0000000000 RZZXEF         .0000000000 RZZXT         .0000000000
   RZZY        -.5635336621 RZZYEF         .0000000000 RZZYT        -.5635336621
   RXYZ         .0000000000 RXYZEF         .0000000000 RXYZT         .0000000000

   Octupole moments in atomic units ( W.R.T. center of mass) 
   OXXX         .0000000000 OXXXEF         .0000000000 OXXXT         .0000000000
   OYYY        2.6846170811 OYYYEF         .0000000000 OYYYT        2.6846170811
   OZZZ        2.8528209341 OZZZEF         .0000000000 OZZZT        2.8528209341
   OXXY        -.4167505651 OXXYEF         .0000000000 OXXYT        -.4167505651
   OXXZ        2.8722565694 OXXZEF         .0000000000 OXXZT        2.8722565694
   OYYX         .0000000000 OYYXEF         .0000000000 OYYXT         .0000000000
   OYYZ       -5.7250775035 OYYZEF         .0000000000 OYYZT       -5.7250775035
   OZZX         .0000000000 OZZXEF         .0000000000 OZZXT         .0000000000
   OZZY       -2.2678665159 OZZYEF         .0000000000 OZZYT       -2.2678665159
   OXYZ         .0000000000 OXYZEF         .0000000000 OXYZT         .0000000000

   Octupole moments in 10**(-34) esu*cm**3 ( W.R.T. center of mass) 
   OXXX         .0000000000 OXXXEF         .0000000000 OXXXT         .0000000000
   OYYY        1.9106097612 OYYYEF         .0000000000 OYYYT        1.9106097612
   OZZZ        2.0303184250 OZZZEF         .0000000000 OZZZT        2.0303184250
   OXXY        -.2965963762 OXXYEF         .0000000000 OXXYT        -.2965963762
   OXXZ        2.0441505334 OXXZEF         .0000000000 OXXZT        2.0441505334
   OYYX         .0000000000 OYYXEF         .0000000000 OYYXT         .0000000000
   OYYZ       -4.0744689583 OYYZEF         .0000000000 OYYZT       -4.0744689583
   OZZX         .0000000000 OZZXEF         .0000000000 OZZXT         .0000000000
   OZZY        -.5635336621 OZZYEF         .0000000000 OZZYT        -.5635336621
   OXYZ         .0000000000 OXYZEF         .0000000000 OXYZT         .0000000000

 1 a.u. = 0.711688 10**(-34) esu*cm**3 

          ---------------------------------------------
          Electrostatic potential/diamagnetic shielding
          ---------------------------------------------

 1 a.u. = 9.07618 esu/cm ( or statvolts ) 

 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs


 No grid defined. 
   Point      X         Y         Z     Potential(a.u.)   Diamagnetic shielding(a.u.)
     1      .00000    .00000    .27143     -21.350336            22.266247
     2      .00000   1.88043   -.83572       -.502543             4.462950
     3      .00000  -1.48043  -1.33572       -.502303             4.457788
     4      .00000    .02238    .11954      35.812867            17.252239

          --------------
          Electric field
          --------------

 1 a.u. = 0.171524 10**(-8) dyn/esu 


 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs

    Atom      X         Y         Z                        Electric field (a.u.)
                                              X              Y              Z           Field
  ------------------------------------------------------------------------------------------------
     1      .00000    .00000    .27143        .000000       -.043506        .291224        .294456
     2      .00000   1.88043   -.83572        .000000        .021786        .020256        .029748
     3      .00000  -1.48043  -1.33572        .000000       -.026318        .013458        .029560

          -----------------------
          Electric field gradient
          -----------------------

 1 a.u. = 0.324123 10**(16) esu/cm**3  ( or statvolts/cm**2 ) = 0.97174 10**(22) v/m**2 


 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs

  Atom      X         Y         Z                            Electric field gradient in molecular frame (a.u.)
                                             XX             YY             ZZ             XY             XZ             YZ
  ------------------------------------------------------------------------------------------------------------------------------
     1      .00000    .00000    .27143       -.841764       -.234492       1.076256        .000000        .000000       -.197529
     2      .00000   1.88043   -.83572        .050716       -.081983        .031267        .000000        .000000        .059935
     3      .00000  -1.48043  -1.33572        .049996       -.038911       -.011085        .000000        .000000       -.080144

  Atom   Principal components of the -EFG- tensor (a.u.)         Asymmetry parameter eta
  --------------------------------------------------------------------------------------
     1       -.263613       -.841764       1.105376                      .523035
     2        .050716        .057096       -.107811                      .059177
     3        .049996        .056345       -.106341                      .059703

 Orientation ( in degrees ) of the principal axis of -EFG- tensor w.r.t absolute (molecular) frame

 Atom =     1
 ------ 
       90.000000        .000000      90.000000
        8.386367      90.000000      98.386367
       81.613633      90.000000       8.386367

 Atom =     2
 ------ 
         .000000      90.000000      90.000000
       90.000000      66.686712      23.313288
       90.000000      23.313288     113.313288

 Atom =     3
 ------ 
         .000000      90.000000      90.000000
       90.000000     130.075936      40.075936
       90.000000      40.075936      49.924064

          ---------------------
          Electron/spin density
          ---------------------

 1 a.u. = 4.80286 10**(-10) esu 

 No grid defined. 

 Read molecular orbitals from ./prop_h2o.movecs


 Read molecular orbitals from ./prop_h2o.movecs


 Total electron density 
 ---------------------- 
   Point      X         Y         Z      Density (a.u.)
     1      .00000    .00000    .27143     193.696190
     2      .00000   1.88043   -.83572        .213402
     3      .00000  -1.48043  -1.33572        .213051
     4      .00000    .02238    .11954      27.464480
 
 
                                NWCHEM Input Module
                                -------------------
 
 

 Summary of allocated global arrays 
 ----------------------------------

   No active global arrays
  

                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  290      304     1335      694      828       12        0        0     
bytes total:             1.94e+05 6.75e+04 8.10e+04 3.84e+02 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 8960 bytes
 



                                     CITATION
                                     --------

            Please use the following citation when publishing results
            obtained with NWChem:

            High Performance Computational Chemistry Group, "NWChem, A
            Computational Chemistry Package for Parallel Computers, 
            Version 2.1" (1997), Pacific Northwest National Laboratory,
            Richland, Washington 99352-0999, USA.



 Total times  cpu:        1.1s     wall:        1.2s
